src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,6 +68,14 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    try {
+      List<YoutubePost> newPost = YoutubeApi.getYoutubePost();
+      response.setContentType("text/html;");
+      response.getWriter().println(newPost);
```

Comment:
nit: this will conflict with the response write at lines 93-94. I'd recommend to remove them for now, since now the posts will go through the sentiment API and won't be returned as is